### ISSUE TITLE
Settings re-ordering / pruning / re-categorization

### ISSUE THREAD

**Issue by fr500** [libretro/RetroArch#6600]
I think it would be important to reorder settings to... something that makes more sense, I keep seeing more and more stuff that doesn't really belong anywhere in some platforms.

I'll be adding screenshots to show what I mean 

![image](https://user-images.githubusercontent.com/1721040/39065994-2b28feae-4499-11e8-93bd-06be62e1f491.png)

![image](https://user-images.githubusercontent.com/1721040/39066105-90dbca4c-4499-11e8-8479-d441ce35daac.png)

![image](https://user-images.githubusercontent.com/1721040/39066168-c93ff732-4499-11e8-80ca-5495b090b8f7.png)

This one seems perfectly fine except for the fact that audio latency was removed even though it's completely related to audio
![image](https://user-images.githubusercontent.com/1721040/39066178-d3b9233c-4499-11e8-8201-2252556c6b2b.png)

This one seems fine too
![image](https://user-images.githubusercontent.com/1721040/39066237-fd0a400e-4499-11e8-95e1-303a303d8764.png)

![image](https://user-images.githubusercontent.com/1721040/39066321-42c27cd8-449a-11e8-8936-4e253e9f26a0.png)

![image](https://user-images.githubusercontent.com/1721040/39066371-75b0bea2-449a-11e8-85a8-efaa77d00a2e.png)

![image](https://user-images.githubusercontent.com/1721040/39066411-956017a2-449a-11e8-9886-06032d3cf1e6.png)

Seems fine
![image](https://user-images.githubusercontent.com/1721040/39066422-9dcb4b96-449a-11e8-9f91-65fde021a1ca.png)

Seems fine too
![image](https://user-images.githubusercontent.com/1721040/39066438-aa85c438-449a-11e8-907f-1e2a3c9cc8eb.png)

Seems fine too
![image](https://user-images.githubusercontent.com/1721040/39066442-b119b818-449a-11e8-848d-7d75a3308141.png)

There are huuuuge menus with 20+ entries with no submenus, this one has 3 and has a submenu with two entries?
![image](https://user-images.githubusercontent.com/1721040/39066504-e946a7dc-449a-11e8-9d14-a921ee616967.png)

Seems fine, maybe make it capture and move the GPU screenshot options there?
![image](https://user-images.githubusercontent.com/1721040/39066530-011d8768-449b-11e8-9b82-294dd9073db9.png)

![image](https://user-images.githubusercontent.com/1721040/39066664-741e4e14-449b-11e8-8e43-05cadc0ab262.png)

![image](https://user-images.githubusercontent.com/1721040/39066687-8676a3b8-449b-11e8-8e09-9a4167a605eb.png)

**Comment by fr500** [comment]
Also, we have too many categories, that's why there is so much overlap here.
Maybe more categories could have sub-categories instead of having so many top level categories

Example:

![image](https://user-images.githubusercontent.com/1721040/39066780-e79ee10a-449b-11e8-87a0-08129924d3f6.png)

![image](https://user-images.githubusercontent.com/1721040/39066803-03d2942a-449c-11e8-911d-07325b01b214.png)
![image](https://user-images.githubusercontent.com/1721040/39066817-110781d2-449c-11e8-9222-05f56767c226.png)

**Comment by hizzlekizzle** [comment]
I agree. If we can hide some of those extraneous options on platforms where they have no effect, we definitely should do so.

Audio latency is more about getting rid of crackles rather than what people typically think of as "latency", so yeah, I think it should go back to audio. Max swapchain could go either way for me. I don't know if people more frequently adjust it down for latency or up for smoother video. /shrug

Network > netplay submenu seems like a slam-dunk to me.

Putting screenshots and recording together makes sense to me, too.

**Comment by i30817** [comment]
Speaking of setting, i was expecting a while ago that just disabling 'GPU Screenshots Enable' would have been enough to get screenshots of the native emulated device native resolution, but this was not the case. I had to disable fullscreen and set 'Windowed Scale' to 1.0 first (though this may have been because PPSSPP was crashing fullscreen).

Not sure if fixable.

**Comment by Ryunam** [comment]
Personally, as I wrote also on the comments section of the pull request I had submitted for the creation of the Latency menu, I disagree with the concerns raised on this topic. Allow me to recap my points in this occasion, bearing in mind that if a consensus is ultimately reached on the matter, I will put my efforts towards moving the options elsewhere.
1)  It is clear that Max Swapchain, Hard GPU Sync and Hard GPU Frames have to do with Video first and foremost, but the reason I placed them under Latency is exactly because they have an effect on Latency during performance. My idea is that it’s much more convenient to have all the options that have a bearing on latency in one place, rather than having them all dispersed between the different categories and forcing the user to navigate back and forth between one Settings page and the other in order to strike an ideal balance for that particular content / core.

2) Similarly, Audio Latency in my experience does affect the immediacy and responsiveness of audio cues in game and it makes sense from a ”tweaker / micromanager” perspective to have it there along with the other latency-affecting configurations. Once again, getting the right balance between how low you can push audio latency, how low you can bring Hard GPU Sync, how high you can raise Frame Delay, etc. is really cumbersome if you place all these options in different places.

3) If compared to what you typically find under Video, Audio and Input - where it’s likely that you set a certain option to a certain value and you leave it like that permanently for all content - all the options I put under Latency often require special and continued tweaking for each core and/or type of content in order to find the right balance.
